src/__tests__/pages/_app.test.js

```diff
@@ -87,4 +90,21 @@ describe('_app.js', () => {
     mount(<App {...mockProps} />)
     expect(themeInChild).toMatchObject(standardTheme)
   })
+
+  it('initializes app with Growthbook and calls it with features', () => {
+    expect.assertions(1)
+    const App = require('src/pages/_app').default
+    const mockProps = getMockProps()
+    const mockGrowthbook = {
+      setFeatures: jest.fn(),
+    }
+    GrowthBook.mockImplementation(() => {
```

Comment:
@kmjennison Wonder if you could help out a little bit with this test. I can't figure out why the actual component doesn't pick up the `mockImplementation` call.

Reply:
@jedtan Fixed in this commit: https://github.com/gladly-team/tab-web/pull/304/commits/241bceec270e1cb23a0bacdad7e5a07ccb4d7fd4

The GrowthBook behavior you're testing happens on module load. By default, Jest doesn't reset modules between tests and so non-component code will only fire on the first test. You can reset modules with `jest.resetModules()`, which we do in that separate test file. We don't do it everywhere because it's unwieldy to require dependencies in every test (and I'd guess it's probably less performant for tests, not sure though).

https://jestjs.io/docs/jest-object#jestresetmodules
https://github.com/facebook/jest/issues/3236